crates/ide_completion/src/tests/pattern.rs

```diff
@@ -444,3 +445,75 @@ fn foo() {
         expect![[r#""#]],
     );
 }
+
+#[test]
+fn completes_associated_const() {
+    check_empty(
+        r#"
+#[derive(PartialEq, Eq)]
+struct Ty(u8);
+
+impl Ty {
+    const ABC: Self = Self(0);
+}
+
+fn f(t: Ty) {
+    match t {
+        Ty::$0 => {}
+        _ => {}
+    }
+}
+"#,
+        expect![[r#"
+            ct ABC const ABC: Self
+        "#]],
+    );
+
+    check_empty(
+        r#"
+struct MyEnum;
```

Comment:
This is still a struct

Reply:
I fixed it, but it looks like the GitHub service was downgraded. It's not updating.